File: src/test/examples/ReturnVoid.lua
function func_void()
    print("I am a void function");
end
func_void();

local function func_local_void()
    print("I am a void local function");
end
func_local_void();

function func_void_has_param(a)
    print(a);
end
func_void_has_param(1);

local function func_local_void_has_param(b)
    print(b);
end
func_local_void_has_param(2);

File: src/test/examples/ReturnVoid.cs
void func_void()
{
    Console.WriteLine("I am a void function");
}
func_void();

void func_local_void()
{
    Console.WriteLine("I am a void local function");
}
func_local_void();

void func_void_has_param(int a)
{
    Console.WriteLine(a);
}
func_void_has_param(1);

void func_local_void_has_param(int b)
{
    Console.WriteLine(b);
}
func_local_void_has_param(2);